### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by plynchnlm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,30 @@
-# Equivalence Classes
+# Equivalence Classes for LOINC Terms (LOINC version 2.66)
 
-These files are associated with the AMIA 2019 Workshop W17: FHIR forms, flowsheets, and other EHR functions for FHIR.
+For some purposes, such a flowsheet, it is useful to regard groups of
```
Comment (lizamos) [inline README.md]: Add "Clinical or EHR" to flowsheet to help clarify.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,30 @@
...
+[LOINC](https://loinc.org/) terms as equivalent.  We call these groups
+"equivalence classes", and have defined a set of rules for generating them.  The
```
Comment (lizamos) [inline README.md]: Where are we defining the rules - will we make the rules available as well?

**Review by lizamos — CHANGES_REQUESTED** [review]
Just one addition. Otherwise, looks good!

File: README.md (hunk 1 above)
Comment (plynchnlm) [reply README.md]: Added.

Comment (plynchnlm) [reply README.md]: Added

**Review by lizamos — APPROVED** [review]

**Pull request merged**